gen/dcos-config.yaml

```diff
@@ -1495,22 +1495,31 @@ package:
         "role": "master",
         "pull": true,
         "diagnostics-bundle-dir": "{{ diagnostics_bundles_dir }}",
-        "endpoint-config": "/opt/mesosphere/etc/dcos-diagnostics-endpoint-config.json",
+        "endpoint-config": [
+          "/opt/mesosphere/etc/dcos-diagnostics-endpoint-config.json",
+          "/opt/mesosphere/etc/dcos-diagnostics-endpoint-config-addition.json"
```

Comment:
I'm not sure I'm following what's going on here. Is the `dcos-diagnostics-endpoint-config-addition.json` for enterprise endpoints? If so, it might be clearer if we name it `-enterprise` instead of `-addition`, unless we think there will be other non-enterprise endpoints that we'll need to add.

Reply:
The purpose of this is to not be included in EE.